semgrep/tests/performance/test_public_repos_perf.py

```diff
@@ -0,0 +1,47 @@
+import subprocess
+from dataclasses import dataclass
+from pathlib import Path
+
+import pytest
+
+
+@dataclass
```

Comment:
Oh, we need to support 3.6, make this namedtuple?

Reply:
Looks like it: